### DISCUSSION TITLE
How to divide Cora datasets？

### DISCUSSION THREAD

**Discussion by Chrisable** [pyg-team/pytorch_geometric#4572] [General]
How to divide the Cora dataset into two parts, and the node index of each part is given rather than random. For example, the first part of the node index is [0,3,5,... 1000], and the second part of the node index is [1,2,4,...]

**Comment by rusty1s** [comment]
If you want to divide a graph by indices, you may want to look into the `Data.subgraph()` functionality:
```python
data1 = data.subgraph(torch.tensor([0, 3, 5, ..., 1000]))
data2 = data.subgraph(torch.tensor([1, 2, 4, ...]))
```

**Reply by Chrisable** [reply]
Thank you for your reply.By the way,when i divide two subgraph,why one subgraph get a  higher accuracy.my model is gcn
![image](https://user-images.githubusercontent.com/49330390/166635102-4e8b2bcf-de05-41b7-a69d-1398b8ca912a.png)

**Reply by Chrisable** [reply]
Shouldn't the accuracy drop when there is less data？

**Reply by rusty1s** [reply]
In general, you are right. However, it can also mean that the nodes in one subgraph are way easier to classify than the nodes in the other one.
